hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -545,6 +545,12 @@
       run_opts: ["+i2c_idx=2"]
       en_run_modes: ["sw_test_mode_test_rom"]
     }
+    {
```

Comment:
i usually recommend adding these tests to CI also.
That way if there's a big block level change (tb / dv agent), we'll know sooner what the top level impact is.

Reply:
Good point @tjaychen , I think this makes sense. I see that @vrozic added this test to the smoke regression list on Line 1743.